crates/atuin-client/src/settings.rs

```diff
@@ -350,6 +350,9 @@ pub struct Daemon {
     /// The path to the unix socket used by the daemon
     pub socket_path: String,
 
+    /// Use a socket passed via systemd's socket activation protocol, instead of the path
+    pub systemd: bool,
```

Comment:
Could we rename this one? I think it's worth making it clear that this is for socket activation in the name. Some people may think that it's for daemonizing with systemd

```suggestion
    pub systemd_socket: bool,
```